bme590_melanoma_detection/validate.py

```diff
@@ -0,0 +1,28 @@
+import base64
+
+#Validate base64 image strings, ie is the string really an image?
+def is_valid_image_string(some_str):
```

Comment:
you can also test this by passing in "hello world" and asserting false, and then passing in real base64 and asserting True. Does this take into account the`data:image/jpeg` type header stuff? It seems like this expects that portion to already be clipped off, which is okay but you may run into problems with the clipping (didn't see where you did that) if the string is like two characters like `"hi"`